src/Api/IdentifiersExtractor.php

```diff
@@ -28,6 +28,8 @@
 /**
  * {@inheritdoc}
  *
+ * @deprecated use ApiPlatform\Metadata\IdentifiersExtractor instead
```

Comment:
This should not go in 3.1 stable as per [semver](https://semver.org/#spec-item-7).